packages/backend/src/modules/tvl/ArbitrumTvl.ts

```diff
@@ -38,13 +52,30 @@ export function createArbitrumTvlSubmodule(
     ChainId.ARBITRUM,
   )
 
+  const arbitrumBalanceProvider = new ArbitrumBalanceProvider(
+    arbitrumClient,
+    multicallClient,
+  )
+
+  const arbitrumBalanceUpdater = new BalanceUpdater(
+    arbitrumBalanceProvider,
+    arbiscanBlockNumberUpdater,
+    db.balanceRepository,
+    db.balanceStatusRepository,
+    clock,
+    config.projects,
```

Comment:
make this empty array for now

Reply:
in the future we will come up wit a per-chain config

Reply:
https://github.com/l2beat/l2beat/pull/1657/commits/937425eb298ac32c49d6068ebad8f90ac7a522b6